lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -60,32 +60,36 @@ def add_copy_resources_script_phase(native_target, spec)
 
             input_paths_by_config = {}
             output_paths_by_config = {}
-            if use_input_output_paths
```

Comment:
this is no longer behind this if statement because we need to calculate the paths to determine if we should remove or add the script phase.